examples/armv4t/gdb/host_io.rs

```diff
@@ -1,5 +1,6 @@
 use std::io::{Read, Seek, Write};
 
+use crate::TEST_PROGRAM_ELF;
```

Comment:
nit: import organization. 

this should be grouped with the other `crate` import.